devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -49,357 +51,364 @@ metadata {
 			tileAttribute ("statusText", key: "SECONDARY_CONTROL") {
 				attributeState "statusText", label:'${currentValue}'
 			}
-        }
+		}
 
 		standardTile("motion", "device.motion", width: 2, height: 2, canChangeIcon: true, canChangeBackground: true) {
 			state "active", label:'motion', icon:"st.motion.motion.active", backgroundColor:"#00A0DC"
 			state "inactive", label:'no motion', icon:"st.motion.motion.inactive", backgroundColor:"#cccccc"
 		}
 		valueTile("temperature", "device.temperature", width: 2, height: 2) {
-			state "temperature", label:'${currentValue}°', unit:"F", icon:"", // would be better if the units would switch to the desired units of the system (imperial or metric)
-			backgroundColors:[
-				[value: 0,  color: "#153591"], // blue=cold
-				[value: 65, color: "#44b621"], // green
-				[value: 70, color: "#44b621"], // green
-				[value: 75, color: "#f1d801"], // yellow
-				[value: 80, color: "#f1d801"], // yellow
-				[value: 85, color: "#f1d801"], // yellow
-				[value: 90, color: "#d04e00"], // red
-				[value: 95, color: "#bc2323"]  // red=hot
-			]
+			state "temperature", label:'${currentValue}°', icon:"",
+				backgroundColors:[
```

Comment:
How does the app or UI differentiate between Celsius and Fahrenheit?

Reply:
It doesn't. The best hack we have (this one) is to have two blue->red ranges that more or less lie in the acceptable indoor temperature ranges for celsius and fahrenheit.